brick.py

```diff
@@ -8,14 +8,16 @@ def __init__(self, position, dimensions):
         self.rect = pygame.Rect(
                 position[0], position[1], dimensions[0], dimensions[1]
             )
-
-        self.position = pygame.Vector2(position[0], position[1])
-        self.speed = 12
-
+        
         self.color = (0, 0, 0)
 
     def update(self):
         self.draw()
 
-    def draw(self):
+    def draw(self, offset = (0, 0)):
+        x, y = self.rect.x, self.rect.y
+        self.rect.x += offset [0]
+        self.rect.y += offset [1]
         pygame.draw.rect(self.gameDisplay, self.color, self.rect)
+        self.rect.x = x
+        self.rect.y = y
```

Comment:
Remove the No new line at EOF.

Aren't these something update should handle before calling draw?

Reply:
this is for drawing with offset if offset is provided